app/views/layouts/application.html.slim

```diff
@@ -8,8 +8,13 @@ html
     == stylesheet_link_tag 'application'
     == javascript_pack_tag 'application'
 
+    script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"
+    script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"
+    script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"
```

Comment:
Technically, Jquery kind of old in 2020, we could go without Jquery, in the real project, we don't use Jquery anymore 😃

Here you can decide to continue using Jquery or just using Vanilla JS, Boostrap has the [Boostrap.Native]( https://thednp.github.io/bootstrap.native/), you can check it out if you don't need to use Jquery :) 

Another thing, we don't make our JS like this, like adding the library by using their CDN directly in the application layout, we build it by our JS file with Webpack stuff...so you could have a look on that...

Reply:
Fixed in 655101f0712287dda82686c95349242499a0167b 1c59e088645e538b97a81acf357b0b3865b426de 6d05f6d174cc9a1b020f95c7f81924dfd300d851